datahub-web-react/src/app/entity/shared/siblingUtils.ts

```diff
@@ -169,35 +169,47 @@ export const shouldEntityBeTreatedAsPrimary = (extractedBaseEntity: { siblings?:
     return isPrimary;
 };
 
-export const combineEntityDataWithSiblings = <T>(baseEntity: T): T => {
-    if (!baseEntity) {
-        return baseEntity;
-    }
-    const baseEntityKey = Object.keys(baseEntity)[0];
-    const extractedBaseEntity = baseEntity[baseEntityKey];
-
+const combineEntityWithSiblings = (entity: any) => {
```

Comment:
Is there a reason we need it as `any` here and can't be a T or an EntityType or something more strongly typed perhaps?